distributed/worker.py

```diff
@@ -4120,3 +4121,33 @@ def gpu_startup(worker):
         return nvml.one_time()
 
     DEFAULT_STARTUP_INFORMATION["gpu"] = gpu_startup
+
+
+def print(*args, **kwargs):
+    """Dask print function
+    This prints both wherever this function is run, and also in the user's
+    client session
+    """
+    try:
+        worker = get_worker()
+    except ValueError:
+        pass
+    else:
+        worker.log_event("print", {"args": args, "kwargs": kwargs})
```

Comment:
I just used this in practice.  I was reminded that we should probably call `str` on all of the args before we send them along msgpack.

Reply:
No problem but what's the reason? We cannot do the same for the warning since some of the args/kwargs could, in theory, by actual warning classes.